project/config/lgbcni/config/views.view.front_news_item.yml

```diff
@@ -343,7 +343,7 @@ display:
       use_more_always: true
       use_more_text: 'More news'
       link_display: custom_url
-      link_url: 'http://lgbc-ni.org.uk.lndo.site/news'
+      link_url: 'http://lgbc-ni.org.uk/news'
```

Comment:
I think if you just leave this as /news it will make it a relative link to the site no matter what the URL is. So it will work locally or in all platform environments.